packages/fxa-email-service/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.141.6
```

Comment:
Hmm, the version in `Cargo.toml` should have changed with this too. Note to self, check the release script isn't broken here.